### PR TITLE
Login

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Login completed
- Many fixes in HTML and CSS

## Screenshot
Captura de Pantalla 2020-11-06 a la(s) 11 15 51 a  m

## Trello Card
https://trello.com/c/ug8AXCwZ/32-login

### CODE REVIEW

File: maquetacion/src/scss/_styles.scss [hunk 1]
```diff
@@ -9,27 +9,27 @@
...
-      position: relative;
-      right: 0;
-      top: 2.5px;
-    }
+  &::before {
+    content: url('../../assets/back-arrow-20.png');
+    padding-right: 10px;
+    position: relative;
+    right: 0;
+    top: 2.5px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]:
```suggestion
    top: 3px;
```

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_styles.scss (hunk 1 above)
Comment (seromarin) [reply maquetacion/src/scss/_styles.scss]: Fixed

**Review by idcuadrowolox — APPROVED** [review]

File: maquetacion/src/scss/_form.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  &__input {
+    border: none;
+    border-radius: 5px;
+    height: 32px;
+  }
+}
+
+.form-divider {
```
Comment (mcallegari10) [inline maquetacion/src/scss/_form.scss]: This class and the `hr` element are not really needed. The way to achieve this without any new element is to add a `border-bottom` to the upper element (in this case, the login button)

**Review by mcallegari10 — CHANGES_REQUESTED** [review]
Just that comment 😄  Great job!

File: maquetacion/src/scss/_form.scss (hunk 2 above)
Comment (seromarin) [reply maquetacion/src/scss/_form.scss]: Solved with a form container

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**